plugins/modules/ndo_service_device_cluster.py

```diff
@@ -0,0 +1,641 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_service_device_cluster
+short_description: Manage Service Device Clusters on Cisco Nexus Dashboard Orchestrator (NDO).
+description:
+- Manage Service Device Clusters on Cisco Nexus Dashboard Orchestrator (NDO).
+- This module is supported on ND v3.2 (NDO v4.4) and later.
+author:
+- Shreyas Srish (@shrsr)
+options:
```

Comment:
Do any of the non required options have default behaviour when not provided? Should we add this to the documentation?

Reply:
ok